keycloak/helm/keycloak/Chart.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v2
+name: keycloak
+description: helm chart for keycloak
+type: application
+version: 0.1.0
+appVersion: "v1.0.0"
+
+dependencies:
+- name: keycloakx
```

Comment:
I think you probably need to wire in the pg credentials into the keycloak app here.